src/CLI/Generator.cs

```diff
@@ -6,155 +6,158 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text;
 using CppAbi = CppSharp.Parser.AST.CppAbi;
 
 namespace CppSharp
 {
     class Generator : ILibrary
     {
-        private Options _options;
-
-        private String _triple = "";
-        private CppAbi _abi = CppAbi.Microsoft;
+        private Options Options = null;
```

Comment:
Private fields start with a lowercase letter. To make a distinction between a field and a parameter, use "this":

this.options = options;